index.js

```diff
@@ -21,27 +22,57 @@ function enableEnterButton (length) {
 
 function outcomeOutput(bool, inputWord) {
     document.querySelector('.textInput').style.display = 'none'
-    document.querySelector('.submitFormButton').style.display = 'none'
     document.querySelector('form').style.justifyContent = 'center'
     let pTag = document.createElement('p')
+    let resultDiv = document.createElement('div')
+    resultDiv.classList.add('textInput', 'retryInput')
     if(bool) {
         let correctText = document.createTextNode(inputWord.toLowerCase() + ' was correct')
         pTag.appendChild(correctText)
-        document.querySelector('form').appendChild(pTag)
+        resultDiv.appendChild(pTag)
+        document.querySelector('form').appendChild(resultDiv)
+        buttonSwitchToRetry()
     } else {
         let incorrectText = document.createTextNode(inputWord.toLowerCase() + ' was incorrect')
         pTag.appendChild(incorrectText)
-        document.querySelector('form').appendChild(pTag)
+        resultDiv.appendChild(pTag)
+        document.querySelector('form').appendChild(resultDiv)
     }
     document.querySelector('form').style.flexDirection = 'row-reverse'
 }
 
+function addTileRow() {
```

Comment:
try where possible not to rely on global vars inside of functions - instead make a param to pass in the global var when the function is called